core/src/main/scala/org/locationtech/rasterframes/extensions/DataFrameMethods.scala

```diff
@@ -155,6 +155,9 @@ trait DataFrameMethods[DF <: DataFrame] extends MethodExtensions[DF] with Metada
   def withPrefixedColumnNames(prefix: String): DF =
     self.columns.foldLeft(self)((df, c) ⇒ df.withColumnRenamed(c, s"$prefix$c").asInstanceOf[DF])
 
+  /**  */
+  def tileStat(): RasterFrameStatFunctions = new RasterFrameStatFunctions(self)
```

Comment:
Working on an alternative approach to discuss, one that takes an approach more in line with the other aggregate functions (i.e. `rf_agg_foo_bar`). While I know that Spark DataFrames uses this extension method approach in a couple of places (`stats`, `na`, ...) I think it adds an inconsistency that can feel arbitrary if the same thing can be achieved with columnar functions. `na` is a good example of one that can't, because it's effectively implementing a builder for NA handling, but I don't think were in that situation.